docs/Apps/Metabolomic Data/Labeled LC-MSMS Workflow.md

```diff
@@ -139,6 +141,86 @@ Click "Run* to plot the fractional enrichment plot with all the labels present a
 
 ![Visualization plot](../../img/LabeledLC-MS2Workflow/VisualizationPlot.png) <center> **Figure 16.** Visualization plot </center>
 
+## Phi<sup>BETA</sup> Tab
+
+This tab enables the user to perform Phi analysis. The term 'Phi' denoted by 'φ' is ratios of fractional enrichments of isotopomers/isotopologues of a product to the fractional enrichments of isotopomers/isotopologues of a precursor in the pathway.
+
+- Consider, ![A to B to C](../../img/LabeledLC-MS2Workflow/phi3.png), then ![Phi calculation formula](../../img/LabeledLC-MS2Workflow/phi2.png), which means if there is **another flux** contributing to the generation of B, then other pathway contributes to **FE**<sub>**B* **</sub> in this case, **𝛷AB < 1**
```

Comment:
@KMaheswari Rephrase this paragraph. Also, fix the bold in the last sentence. The </sub> class should not have two * in front of it.